src/mapgen_v7.cpp

```diff
@@ -59,7 +59,7 @@ MapgenV7::MapgenV7(int mapgenid, MapgenV7Params *params, EmergeManager *emerge)
 	this->large_cave_depth    = params->large_cave_depth;
```

Comment:
I tried removing these `this-> .. = params-> ..` lines as requested but oddly this caused mapgen to completely fail, nothing generated.